tests/unit/test_diffsync.py

```diff
@@ -898,3 +884,147 @@ class NoDeleteInterfaceDiffSync(BackendA):
     diff = extra_models.diff_from(backend_a)
     print(diff.str())  # for debugging of any failure
     assert not diff.has_diffs()
+
+
+def test_diffsync_tree_traversal(backend_a):
+    assert backend_a.get_tree_traversal(True) == {"site": {"device": {"interface": {}}, "person": {}}, "unused": {}}
+    text = "BackendA\n├── site\n│   ├── device\n│   │   └── interface\n│   └── person\n└── unused"
```

Comment:
```suggestion
    text = """\
BackendA
├── site
│   ├── device
│   │   └── interface
│   └── person
└── unused"""
```

Reply:
I'm not a fan of either of these views, but I guess lesser of 2 evils.